index.html

```diff
@@ -0,0 +1,417 @@
+                        is_correct: true
+                    },
+                    {
+                        id: 3,
+                        text: "Java",
+                        is_correct: false
+                    },
+                    {
+                        id: 4,
+                        text: "c++",
+                        is_correct: false
+                    }
+                ]
+            },
+            {
+                id: 9,
+                text: "JavaScript was originally called ?",
+                answers: [{
+                        id: 1,
+                        text: "Java",
+                        is_correct: false
+                    },
+                    {
+                        id: 2,
+                        text: "Mocha",
+                        is_correct: true
+                    },
+                    {
+                        id: 3,
+                        text: "JSON",
+                        is_correct: false
+                    },
+                    {
+                        id: 4,
+                        text: "c++",
+                        is_correct: false
+                    }
+                ]
+            },
+        ]
+
+        const startBtn = document.querySelector('#start-btn');
+        const nextBtn = document.querySelector('#next-btn');
+        const exitBtn = document.querySelector('#exit-btn');
+        const question = document.querySelector('.question');
+        const answers = document.querySelector('.answers');
+        const counterDiv = document.querySelector('#counter');
+        const choices = document.querySelectorAll('.choice');
+        const quizView = document.querySelector('#quiz-view');
+        const thanksView = document.querySelector('#thanks-view')
+        const alertTxt =
+            "Welcome to JS quiz\n Here are the rules: \n 1 -Don't select the same answer twice.\n2- If all your answers are wrong, you will not get a feedback abput how many question(s) ypu answered.\n3- If your answer is correct, ypu will see green color\n 4- If your answer is wrong, you will see red color";
+        //to keep track of the how many answers were correct
+        let total = 0;
+        //to keep track of the question's number
+        let questionCounter = 0;
+
+        function toggleHidden(e) {
+            
+            if (this === startBtn) {
+                this.classList.toggle('hidden');
+                answers.classList.toggle('hidden');
+                question.classList.toggle('hidden');
+                showQuestion();
+            } else if (this === exitBtn){
+                quizView.classList.toggle('hidden');
+                thanksView.classList.toggle('hidden');
+            }
+        }
+
+        function showTheResult() {
+            const p = document.createElement('p');
+            // let p = document.createElement('p');
+            const text = document.createTextNode(`You got ${total} correct(s) from 9`);
```

Comment:
Choose a more descriptive name.
```suggestion
            const scoreText = document.createTextNode(`You got ${total} correct(s) from 9`);
```